uranium/executables.py

```diff
@@ -69,7 +70,7 @@ def main(build):
         try:
             out, err = popen.communicate()
         except KeyboardInterrupt:
-            os.kill(popen.pid)
+            os.kill(popen.pid, signal.SIGTERM)
```

Comment:
nice catch! but I'm thinking we can use SIGINT vs SIGTERM.

SIGINT is what the KeyboardInterrupt effectively amounts to, and it gives the process time to cleanup. SIGTERM kills the process directly from the OS, allowing leaking of any resources that weren't cleaned up.

https://www.quora.com/What-is-the-difference-between-the-SIGINT-and-SIGTERM-signals-in-Linux